util/config.go

```diff
@@ -39,6 +39,7 @@ func AddGlobalOptions(cmd *mybase.Command) {
 	cmd.AddOption(mybase.StringOption("docker-cleanup", 0, "NONE", `With --workspace=docker, specifies how to clean up containers (valid values: "NONE", "STOP", "DESTROY")`))
 	cmd.AddOption(mybase.BoolOption("reuse-temp-schema", 0, false, "Do not drop temp-schema when done"))
 	cmd.AddOption(mybase.BoolOption("debug", 0, false, "Enable debug logging"))
+	cmd.AddOption(mybase.BoolOption("ignore-my-cnf", 0, false, "Skips reading users's ~/.my.cnf file when determining configuration" ))
```

Comment:
Travis CI is failing on `gofmt` check -- I think it's because of the extra space before closing parens on this line